dashboard/app/models/artifact.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: artifacts
+#
+#  id                 :integer          not null, primary key
+#  name               :string(255)
+#  description        :text(65535)
+#  learning_module_id :integer
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+# Indexes
+#
+#  index_artifacts_on_learning_module_id  (learning_module_id)
+#
+
+class Artifact < ActiveRecord::Base
```

Comment:
It's not obvious looking at this file that this is a PLC artifact or how to find the overview wiki page that you wrote.

Could you move the PLC files into a subdirectory so that they are neatly grouped together and add a README.md in that subdirectory that summarizes or links to the documentation you've written?

Also, please add file comments summarizing the role of each model.  Doesn't need to too long since you have external documentation.